gargoyle_osx.sh

```diff
@@ -132,6 +132,8 @@ cp fonts/Gargoyle*.ttf $BUNDLE/Resources/Fonts
 
 codesign -s - -f --deep Gargoyle.app
 
+exit
+
```

Comment:
What’s this?

Reply:
Accidental commit from testing; I realized I left it here, came to fix it, and saw you beat me by 10 minutes!